packages/manager/modules/manager-components/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@ovh-ux/manager-components",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A set of reusable UI components, to be used across the OVH Manager",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/manager-components"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "lodash": "^4.17.15"
+  },
+  "peerDependencies": {
+    "@ovh-ux/ui-kit": "^4.4.3",
+    "@uirouter/angularjs": "^1.0.23",
```

Comment:
I don't see this dependency used by your module, you can remove this then

Reply:
Done. Thanks :)